src/pkg/notifier/handler/notification/scan_image_handler.go

```diff
@@ -1,12 +1,16 @@
 package notification
 
 import (
+	"context"
 	"time"
 
+	o "github.com/astaxie/beego/orm"
+	"github.com/goharbor/harbor/src/api/artifact"
```

Comment:
Currently, the package pkg calls code of package api, this is against our program model. 
Let's move the event handler into `src/api/event` after the PR https://github.com/goharbor/harbor/pull/11010 merged

Maybe we should rename the package api to a better name

Reply:
Out of the scope this PR. I will submit another PR for this issue.